backend/src/db-utils.js

```diff
@@ -528,6 +528,24 @@ const queryPartnerUniqueUserAmount = async () => {
   );
 };
 
+// bridge query
+const queryBridgeTokenHolder = async (accountId) => {
```

Comment:
You seem to want to have a list of accounts (holders) which minted tokens through bridge, right? If that is the case:

1. the name has to be plural (`queryBridgeTokenHolders`)
2. `accountId` in the arguments misleads a reader; I suggest `bridgeTokenContractId`